recipes/collada-dom/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "collada-dom" %}
+{% set version = "2.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rdiankov/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 3be672407a7aef60b64ce4b39704b32816b0b28f61ebffd4fbd02c8012901e0d
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - boost
+    - pcre
+    - libxml2
+    - zlib
+  run:
+    - boost
+    - pcre
+    - libxml2
+    - zlib
+
+test:
+  requires:
+    - pkg-config
+  commands:
+    - pkg-config --modversion {{ name }}
+
+about:
+  home: http://github.com/rdiankov/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
You need to list the file in https://github.com/rdiankov/collada-dom/tree/master/licenses here.

Also I'm not 100% from the github repo whether this repo is a repackaging of a subset on sourceforge or a mirror of that or a name collision.

Reply:
@xhochy I've fixed the license by putting the file which collects all the licenses. 
Concerning the sources, I've used the one that is distributed along many distributions for Linux and Mac OS X

Reply:
@xhochy By the way, how can I provide several licence files names in this section `license_file`?

Reply:
You can add multiple files in a list @jcarpent, for example:
```
  license: MIT AND BSD-3-Clause
  license_file:
       - LICENSE.txt
       - COPYING.txt
```

Reply:
Thanks a lot!